lib/rdstation/error/format.rb

```diff
@@ -1,9 +1,13 @@
+# frozen_string_literal: true
+
 module RDStation
   class Error
     class Format
-      FLAT_HASH = 'FLAT_HASH'.freeze
-      HASH_OF_ARRAYS = 'HASH_OF_ARRAYS'.freeze
-      ARRAY_OF_HASHES = 'ARRAY_OF_HASHES'.freeze
+      FLAT_HASH = 'FLAT_HASH'
```

Comment:
Por que removeu os `.freeze`?

Reply:
Rodei o correct do Rubocop e ele removeu:
`Style/RedundantFreeze: Do not freeze immutable objects, as freezing them has no effect.`